tests/integration/fixtures.py

```diff
@@ -403,6 +413,51 @@ def _cleanup_changesets(changesets: List[str]) -> None:
 # Helpers for Cfn
 
 
+@pytest.fixture
+def deploy_cfn_template(
+    cfn_client,
+    lambda_client,
+    cleanup_stacks,
+    cleanup_changesets,
+    is_change_set_created_and_available,
+    is_stack_created,
+):
+    stack_name = f"stack-{short_uid()}"
+    change_set_name = f"change-set-{short_uid()}"
+    state = {}
+
+    def _deploy(template_body_or_file, **kwargs):
```

Comment:
In this case I would prefer changing this to a more explicit & properly typed signature (also avoiding mixing body string and file paths). I can add these in a follow-up PR though.